### PR TITLE
Fixing some sun properties to give them quantities.

### PR DESCRIPTION
**Pull request by wafels**
There were some sun properties that did not have astropy quantities attached.  This PR fixes that and some PEP8 stuff.

### CODE REVIEW

File: sunpy/sun/sun.py [hunk 1]
```diff
@@ -31,40 +31,42 @@
...
+           , "equation_of_center"
+           , "geometric_mean_longitude"
+           , "carrington_rotation_number"
+           , "mean_anomaly"
+           , "longitude_Sun_perigee"
+           , "mean_ecliptic_longitude"
+           , "eccentricity_SunEarth_orbit"
+           , "position"
+           , "solar_semidiameter_angular_size"
+           , "solar_cycle_number"]
```
Comment (dpshelio) [inline sunpy/sun/sun.py]: That one is new... wouldn't be better to have the commas on the end?

**Comment by nabobalis** [comment]
This only failed due to the doc build. That is now fixed.

File: sunpy/sun/sun.py (hunk 1 above)
Comment (Cadair) [inline sunpy/sun/sun.py]: yeah commas on the end is more normal...

Comment (nabobalis) [inline sunpy/sun/sun.py]: Do we change this or should I merge?

Comment (Cadair) [inline sunpy/sun/sun.py]: I would like to see it changed.

Comment (nabobalis) [inline sunpy/sun/sun.py]: #BOoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo

**Pull request merged**